### PR TITLE
downgrade --tighten

### PR DESCRIPTION
**Pull request by sigurdm**
Fixes https://github.com/dart-lang/pub/issues/4241

### CODE REVIEW

File: lib/src/entrypoint.dart [hunk 1]
```diff
@@ -1244,4 +1247,139 @@ See https://dart.dev/go/sdk-constraint
       }
     }
   }
+
+  /// Returns a list of changes to constraints ofg workspace pubspecs updated to
```
Comment (mosuem) [inline lib/src/entrypoint.dart:1251]:
```suggestion
  /// Returns a list of changes to constraints of workspace pubspecs updated to
```

File: lib/src/entrypoint.dart [hunk 2]
```diff
@@ -1244,4 +1247,139 @@ See https://dart.dev/go/sdk-constraint
...
+  ///  have their lower bound match the version in [packageVersions] (or
```
Comment (mosuem) [inline lib/src/entrypoint.dart:1252]:
```suggestion
  /// have their lower bound match the version in [packageVersions] (or
```

File: lib/src/entrypoint.dart [hunk 3]
```diff
@@ -1244,4 +1247,139 @@ See https://dart.dev/go/sdk-constraint
...
+  ///  `this.lockFile`).
```
Comment (mosuem) [inline lib/src/entrypoint.dart:1253]:
```suggestion
  /// `this.lockFile`).
```

**Review by mosuem — COMMENTED** [review]
That was fast!

**Review by jonasfj — APPROVED** [review]

**Pull request merged**